content/en/metrics/open_telemetry/_index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Send Metrics from OpenTelemetry to Datadog
+kind: documentation
+further_reading:
+    - link: 'tracing/trace_collection/open_standards/'
+      tag: 'Documentation'
+      text: 'Learn more about OpenTelemetry'
+---
+
+## Overview
+
+[OpenTelemetry][1] (OTel) is an open source observability framework that provides IT teams with standardized protocols and tools for collecting and routing telemetry data. Created as an incubator project by the [Cloud Native Computing Foundation][2] (CNCF), OTel provides a consistent format for instrumenting, generating, gathering, and exporting application telemetry data—namely metrics, logs, and traces—to monitoring platforms for analysis and insight.
+
+## Setup
+
+To send OTel metrics to Datadog, you have two options: the Datadog Agent, or the OTel Collector. Using the Datadog Agent enables you to keep using all [Agent functionalities][3]. For a more vendor-agnostic setup, use the OTel Collector.
+
+### Datadog Agent setup
+
+**OTel SDKs/Libraries** -> **Datadog Agent** -> **Datadog**
+
+#### Configuration
+
+1. [Instrument your system with OpenTelemetry][4].
```

Comment:
Please link the text to this webpage instead: https://opentelemetry.io/docs/instrumentation/